src/styles/Colors.js

```diff
@@ -1,7 +1,7 @@
 export default Colors = {
-  primary: '#F4F4F4',
-  secondary: '#FFFFFF',
-  highlight: '#FF5553',
+  primary: '#89bdd3',
```

Comment:
Haha it’d be cool to add that as a setting. How many apps do that?! But I think at some point we need Isabelle or someone to give us a design review and then revamp the style 😎